libraries/AP_HAL_ChibiOS/hwdef/thepeach-k1/hwdef-bl.dat

```diff
@@ -0,0 +1,84 @@
+# MCU class and specific type
+MCU STM32F4xx STM32F427xx
+
+# USB setup
+USB_PRODUCT 0x0001
+USB_STRING_MANUFACTURER "ThePeach"
+USB_STRING_PRODUCT "FCC-K1 BL"
+
+# board ID for firmware load
+APJ_BOARD_ID 212
+
+# crystal frequency
+OSCILLATOR_HZ 24000000
+
+# ChibiOS system timer
+STM32_ST_USE_TIMER 5
+
+# flash size
+FLASH_SIZE_KB 2048
+
+# location of application code
+FLASH_BOOTLOADER_LOAD_KB 16
```

Comment:
Since this is a 2Mb board I would recommend using 32k bootloader

Reply:
needs to match px4, and 16k is what we use on other F427 boards, 16k is fine